tools/rdiff-backup.spec.template

```diff
@@ -52,7 +52,7 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.pyc
 %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.so
 %ghost %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.pyo
-%doc CHANGELOG COPYING FAQ.html examples.html README.md DEVELOP.md
+%doc CHANGELOG COPYING FAQ.md examples.md README.md DEVELOP.md
```

Comment:
I assume that it should read with `docs/FAQ.md` etc...

Reply:
I'm not sure, as it is in the build directory, and the setup.py moves files around before the copy.

I will do some tests on this before I resubmit it.

Reply:
@ericzolf You were correct.  I'll fix it up.